packages/block-editor/src/components/block-alignment-control/use-available-alignments.js

```diff
@@ -48,5 +52,10 @@ export default function useAvailableAlignments( controls = DEFAULT_CONTROLS ) {
 			availableAlignments.includes( control )
 	);
 
-	return enabledControls;
+	return {
+		enabledControls,
+		layout,
+		// TODO check if this is the check we need to show the `doesn't support wide alignments` text.
+		wideAlignmentsSupport: layout.alignments || wideControlsEnabled,
```

Comment:
I'm not sure if this is the right calculation for supported wide alignments. Folks with theme experience could help.. maybe @carolinan @aristath , @kjellr , @youknowriad

Reply:
I'm not so familiar with the code behind the alignments, but it is similar to the check inside alignment component. I don't see any problems when I remove the layout setting from theme.json.